src/Nethermind/Nethermind.Evm.Test/Tracing/EstimateGasTracerTests.cs

```diff
@@ -190,5 +190,17 @@ public void Throw_operation_canceled_after_given_timeout()
 
             Assert.Throws<OperationCanceledException>(() => tracer.CalculateEstimate(transactionMock));
         }
+
+        [Test]
+        public void Does_not_throw_if_cancellation_token_is_default()
+        {
+            CancellationToken cancellationToken = default(CancellationToken);
+            var tracer = new EstimateGasTracer(cancellationToken);
+            Transaction transactionMock = Substitute.For<Transaction>();
+            
+            Thread.Sleep(TimeSpan.FromSeconds(2));
```

Comment:
In tests - best if we keep Thread.Sleep to minimum - here it will be enough to Sleep for 100ms. Each test that has a long Sleep value contributes  to overall testing times. 30 tests like this would add a minute to test runs.